# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../util-linux/managed.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Injected from Fedora Core CVS for HAL support.

  - Rene Rebe <rene@exactcode.de>

--- util-linux-2.12a/mount/mount.c.sopwith	2004-03-04 20:28:22.000000000 -0500
+++ util-linux-2.12a/mount/mount.c	2004-07-16 16:50:18.792814782 -0400
@@ -191,6 +191,7 @@
   { "diratime",	0, 1, MS_NODIRATIME },	/* Update dir access times */
   { "nodiratime", 0, 0, MS_NODIRATIME },/* Do not update dir access times */
 #endif
+  { "managed", 0, 0, MS_COMMENT },			/* Silently remove this option */
   { NULL,	0, 0, 0		}
 };
 
